stackslib/src/net/tests/download/nakamoto.rs

```diff
@@ -444,10 +443,22 @@ fn test_nakamoto_unconfirmed_tenure_downloader() {
         let mut utd = NakamotoUnconfirmedTenureDownloader::new(naddr.clone(), Some(tip_block_id));
         assert_eq!(utd.state, NakamotoUnconfirmedDownloadState::GetTenureInfo);
 
-        utd.confirmed_aggregate_public_key =
-            Some(agg_pubkeys.get(&tip_rc).cloned().unwrap().unwrap());
-        utd.unconfirmed_aggregate_public_key =
-            Some(agg_pubkeys.get(&tip_rc).cloned().unwrap().unwrap());
+        utd.confirmed_signer_keys = Some(
```

Comment:
This is a total nit (epsecially since this is a test), but it prob is slightly faster to just do this lookup once and then assign `utd.uncornfirmed_signer_keys = utd.confirmed_signer_keys.clone();`